File: webapiangular2/webapiangular2/Models/Item.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace webapiangular2.Models
{
    public class Item
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public double Price { get; set; }
        public string Image { get; set; }
        public int Quantity { get; set; }
    }
}

File: app/shop/models/item.ts
import { Review } from "../../shop/models/Review";

export class Item {
    
    Id: number;
    Name: string;
    Price: number;
    Quantity: number;
    Image: string;
    Total: number;
    Reviews: Review[];
    Rate: number;
    constructor(){
        
    }
    
}